### PR TITLE
Minor Fixes and Updates

### PR DESCRIPTION
**Pull request by Jacques00**
* New Shou busts for the ganrael and Kyris.
* New Jacques00 Busts for gray goo, Horace Decker and Penny.
* Bunch of typo fixes.
* Add timed delay for printing cumflation leaking messages.
* New Item: Anusol, for plumping up anuses, by Couch (yet to be implemented to a vendor).

### CODE REVIEW

File: classes/Creature.as [hunk 1]
```diff
@@ -15951,7 +15958,20 @@
...
+				kGAMECLASS.eventBuffer += ParseText(notice);
+				
+				if(amountStored >= (25000 / 4 / 2))
+				{
+					var delayTime:int = 30 * Math.floor(amountStored / 25000);
+					if(delayTime > 240) delayTime = 240;
+					if(delayTime < 30) delayTime = 30;
+					createStatusEffect("Omit Cumflation Messages", 0, 0, 0, 0, true, "Icon_Sperm_Hearts", "Cumflation messages are currently disabled.", false, delayTime);
+				}
+			}
```
Comment (Fenoxo) [inline classes/Creature.as]: Not sure how I feel about this. If people don't like constantly cascading cum down their thighs, maybe they should just press the big, shiny "rest" button and be done with it.

File: includes/mhenga/penny.as [hunk 2]
```diff
@@ -71,15 +71,62 @@ public function pennyBustDisplay(nude:Boolean = false):String
...
 	var hasNude:Boolean = true;
-	if(kGAMECLASS.gameOptions.configuredBustPreferences["PENNY"] != "GATS") hasNude = false;
+	if(!InCollection(kGAMECLASS.gameOptions.configuredBustPreferences["PENNY"], "GATS", "JACQUES")) hasNude = false;
 	
 	var sBust:String = "PENNY";
-	if(nude && hasNude) sBust += "_NUDE";
+	if(nude && hasNude)
+	{
+		sBust += "_NUDE";
+		if(kGAMECLASS.gameOptions.configuredBustPreferences["PENNY_NUDE"] == "JACQUES")
```
Comment (Fenoxo) [inline includes/mhenga/penny.as]: So if I'm reading this, the cool new variants are only possible if Jacques00 is set as your preference for Penny?

Hrmmm

Comment (Jacques00) [inline includes/mhenga/penny.as]: For the moment, yes  D :
I think if the bust options are automatic or prioritize Jacques00, it should auto choose the proper bust... maybe. But yea, super finicky.

File: classes/Creature.as (hunk 1 above)
Comment (Jacques00) [inline classes/Creature.as]: Tru, but it does get tedious with mega-huge loads that don't dissipate after four hours, heheh. This is for the huge loads--I think smaller quantities should show the messages proper. I only did some light testing, so feel free to edit or cut if it doesn't play out to your liking.

**Pull request merged**